jwst/pixel_replace/pixel_replace.py

```diff
@@ -605,10 +656,12 @@ def custom_slice(self, dispaxis, index):
         elif dispaxis == self.VERTICAL:
             return np.s_[index, :]
         else:
-            raise Exception
+            raise IndexError("Custom slice requires valid dispersion axis specification!")
 
     def profile_mse(self, scale, median, current):
-        """Function to feed optimization routine
+        """
+        Create function to provide to optimization routine.
```

Comment:
This function does not appear to return a function, so I find this confusing.  Maybe something like
```suggestion
        Compute MSE of profile.
        
        This function is the minimization criterion provided to the optimization routine.
```

Reply:
Good catch, I let autopilot take over a bit too much here.